views/admin/moduleEdit.pug

```diff
@@ -128,6 +128,11 @@ block content
       label(for="secondary_title") Secondary Title:
       input#secondary_title.form-control(type='text', value="")
     
+    div.d-flex.flex-column.mb-4
+      label(for="subject") Subject(eg. "econ", "physics", "chem"):
```

Comment:
Maybe add a "none" within the e.g. cause some of the modules won't have subjects or don't need